lib/rubygems/security.rb

```diff
@@ -461,9 +469,23 @@ def self.create_digest(algorithm = DIGEST_NAME)
   ##
   # Creates a new key pair of the specified +length+ and +algorithm+.  The
   # default is a 3072 bit RSA key.
+  #
 
-  def self.create_key(length = KEY_LENGTH, algorithm = KEY_ALGORITHM)
-    algorithm.new length
+  def self.create_key(algorithm)
```

Comment:
This is a public method, and we should expect that there's code out there that calls `Gem::Security.create_key` in its current form, using any permutation of the existing parameters.

Perhaps we should define a new method and call that one in the `cert` command instead?

Reply:
I don't think this is an issue because we changed all occurrences in the codebase and I assume it isn't likely for someone else to depend on it. Let's see what the people from RubyGems say.